.gitignore

```diff
@@ -173,3 +173,23 @@ test/data/*.json
 test/data/pipeline/*.json
 test/data/filters/*.json
 test/data/plang/*.json
+*.png
+*.txt
+*.js
+*.ico
+*.pdf
+*.svg
+*.woff
+*.otf
+*.eot
+*.ttf
+*.html
+*.ai
+*.woff2
+*.css
+*.doctree
+doc/_build/html/.buildinfo
+*.jpg
+*.pickle
+doc/_build/html/_static/dummy
+*.inv
```

Comment:
I'm fairly certain we don't want to ignore most of these, but we do want to ignore the contents of `doc/_build` as that is what the documentation builds.